### DISCUSSION TITLE
Is there a current list of tags you use?

### DISCUSSION THREAD

**Discussion by brimwats1** [BryanJenksCommunity/FAQ#71] [Q&A] [answered]
The last one I have looks like this!

```
#📥️ Seed box | items that i am / will be actively working on
#🌱️ Seedlings | distilled from literature notes
#🌞️ Incubator | items not yet ready for planting or in need of planting
#🌲️ Evergreen | forest notes

#✅️ Items that have tasks that i need to complete
#🗺️ Maps of Content (the emerging organizational structure)
#⚙️ General utilities i use in this system
✏️ Workbench
♻️ Flows
🔌️ Plugins
🛑 Thinking About That
#❗️ IMPORTANT
#🏷️ Tag Notes
🔗️☁️

@ Person
{ Books, eBooks, or Audio Books
! Tweets
% Podcasts / Audio media
+ YouTube Videos
( Web Articles or Publications, Newspapers, etc.
& Research Paper
- Thoughts


```

**Comment by tallguyjenks** [comment, marked as answer]
Hey Bri!

my list is usually [HERE](https://publish.obsidian.md/bryan-jenks/Tag+Taxonomy) but there's more that I don't really break out in an indexed fashion

I've also recently updated my home page too: [INDEX](https://publish.obsidian.md/bryan-jenks/INDEX)

and here's a screen shot of my tag pane if you wanted to ask about a specific one. Many tags are also picked up from my CRM and Ancestry vaults too so there's also some clutter:

![image](https://user-images.githubusercontent.com/29872822/115640468-e3de7580-a2cb-11eb-90f3-cc8dbf9e7eff.png)
![image](https://user-images.githubusercontent.com/29872822/115640481-e9d45680-a2cb-11eb-9349-38dec2de4347.png)

**Reply by kenanmike** [reply]
Since this is the most popular q on here, makes more sense to put my follow up q here so that others will see it. 
How do you quickly search for the tags with the emojis? I’d have thought it’s much longer than typing a different character like ( { etc. but you must have a quickish way of doing it.

**Reply by tallguyjenks** [reply]
@kenanmike alfred snippets :)

**Comment by brimwats1** [comment]
Thanks so much Bryan!!

**Comment by kenanmike** [comment]
Nice! I have Alfred power pack too so maybe I’ll try this method. Can you give me an example of a good shortcut? I’m thinking of something like !book for a book emoji?

**Reply by tallguyjenks** [reply]
i do comma snippets because there is always a space after normal comma usage so when there isn't its deliberate and only requires a single keystroke. 

my snippet `,seed` produces: `📝️/🌱️` and if its not processed then my snippet `,rsq` (think **R**ed **SQ**uare) produces `/🟥️` all together `,seed` + `,rsq` = `📝️/🌱️/🟥️`

**Comment by carlotrimarchi** [comment]
I've seen you mentioning `workbench` notes in many videos, but couldn't really find an explanation for it. Are they related to the Workbench plugin? Could you expand on their use case (or point to a video, if there is one)?

**Reply by tallguyjenks** [reply]
I dont really use the workbench plugin per-se but I have a note called workbench i just use as a scratch pad

**Comment by kenanmike** [comment]
Oo yes, I second this request


On 31 Aug 2021, at 19:00, Carlo ***@***.***> wrote:

﻿

I've seen you mentioning workbench notes in many videos, but couldn't really find an explanation for it. Are they related to the Workbench plugin? Could you expand on their use case (or point to a video, if there is one)?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FBryanJenksCommunity%2FFAQ%2Fdiscussions%2F71%23discussioncomment-1260620&data=04%7C01%7C%7C82e195791e304e3b859708d96c98620c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637660224011738104%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=zbpl%2F01wCYrEic8PDP4hiUa6HHC4q2QFgHOh%2F2XdAkM%3D&reserved=0>, or unsubscribe<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAULLIMKAQBFRFSSKIMW3DOTT7T375ANCNFSM43HINF5Q&data=04%7C01%7C%7C82e195791e304e3b859708d96c98620c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637660224011748057%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=545G10FCEsC6hiyG4qp%2F4V4R2UrEhpJZgn8UHLrqPRY%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7C%7C82e195791e304e3b859708d96c98620c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637660224011748057%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=yQTwmtKF%2BNvyegxWHHHd8fS1tAjzcliNZKbJbEaaDwM%3D&reserved=0> or Android<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7C%7C82e195791e304e3b859708d96c98620c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637660224011758017%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=zyzLpBBraqD2P2ctq2rvBHnLPEkdUxddaTvQfxBR5%2F4%3D&reserved=0>.
